src/core/shared_memory_manager.h

```diff
@@ -32,6 +32,25 @@
 #include "src/core/model_config.pb.h"
 #include "src/core/status.h"
 
+/// A struct that records the shared memory regions registered by the shared
```

Comment:
This is outside of any namespace. Move it into shared_memory_manager.cc and put it in anonymous namespace?

Reply:
Moved into the SharedMemoryManager class.